src/components/welcome/MyAccounts/AccountItem.tsx

```diff
@@ -39,10 +35,27 @@ const AccountItem = ({ onLinkClick, chainId, address, ...rest }: AccountItemProp
   const isCurrentSafe = chainId === currChainId && sameAddress(safeAddress, address)
   const trackingLabel =
     router.pathname === AppRoutes.welcome.accounts ? OVERVIEW_LABELS.login_page : OVERVIEW_LABELS.sidebar
+  const isWelcomePage = router.pathname === AppRoutes.welcome.index || router.pathname === AppRoutes.welcome.socialLogin
```

Comment:
This can be removed because we don't have a SafeList on the Welcome page anymore.

Reply:
Ah nice catch! We have to change it to `AppRoutes.welcome.accounts` since we are displaying them there